doc/funcref.sgml

```diff
@@ -4108,6 +4110,28 @@ be used in presence of a prototype.
 </quote>
 
 
+<sect1>get_tv<label id="get_tv"><p>
+
+<quote>
+<descrip>
+<tag/Function/The function returns the system's vertical blank frequency.
+<tag/Header/<tt/<ref id="apple2.h" name="apple2.h">/
```

Comment:
There are quite some other headers containing this prototype. Yes, you are supposed to list them all, see i.e. https://cc65.github.io/doc/funcref.html#ss3.295